pennydjango/config/settings.py

```diff
@@ -43,7 +43,7 @@
 BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
 REPO_DIR = os.path.dirname(BASE_DIR)
 
-APP_NAME = 'Homenet'
+APP_NAME = 'Push Forward'
```

Comment:
Doesn't changing this generate migrations for all the ChoiceFields that use it in the name of the choices?

Reply:
oh right, migrations. my b